kouncil-frontend/src/app/globals.ts

```diff
@@ -0,0 +1,36 @@
+import {Injectable} from '@angular/core';
+import {Server} from './navbar/navbar.component';
+import {HttpClient} from '@angular/common/http';
+
+@Injectable()
+export class Globals {
```

Comment:
Jakoś bym to kurczaczki inaczej nazwał, żeby bardziej przystawało do tego co się faktycznie tu dzieje, bo czuję w takich klasach pokusę na worek, w którym za chwilę wszystko będzie

Reply:
Prawda, zmienię na Servers, żeby nie było wątpliwości